pkg/api/query/v1_test.go

```diff
@@ -1575,6 +1575,7 @@ func TestRulesHandler(t *testing.T) {
 			Name:                      "3",
 			LastEvaluation:            time.Time{}.Add(3 * time.Minute),
 			EvaluationDurationSeconds: 12,
+			KeepFiringFor:             5,
```

Comment:
How do we decide these values?

Reply:
I have given random values based on the other prameters. Also referred the [test cases](https://github.com/prometheus/prometheus/pull/11827/files#diff-d489ad80f3c2e3bbd19602b55dc04c7b0cfd598dc0844a954f6cf6faf1069b5f) at prometheus side; the value is not random i.e they have given 0 for almost every case.

Reply:
What's the range for this value? i.e. what could be possible min/max values for this variable?